articles/connections/passwordless/regular-web-app-email-code/v8/index.md

```diff
@@ -52,6 +52,8 @@ You can follow any of the [Regular Web App Quickstarts](/quickstart/webapp) to s
 
 ### Use your own UI
 
+<%= include('../../../../_includes/_version_warning_auth0js') %>
```

Comment:
`version-warning` can be used only under a version dropdown (that is, at the top of the page).
If the warning is in the middle of the doc you have to use a `warning`

Reply:
This is not the only place so please go through all the files and double check